app/assets/stylesheets/application.css.scss

```diff
@@ -1,4 +1,5 @@
 /*
 *= require_tree .
 *= require_self
+*= require select2
```

Comment:
move this above require tree

Reply:
first we go for the specific ones and then go towards including rest of the js files.